lib/gradle-wrapper.ts

```diff
@@ -24,7 +25,10 @@ export async function getClassPathFromGradle(
   const args = getGradleCommandArgs(targetPath);
   try {
     const output = await execute(gradlePath, args, { cwd: targetPath });
-    return output.trim();
+    const lines = output.trim().split(EOL);
```

Comment:
What's the reason to do that?

Reply:
In my android tests Gradle outputs more than one line, this is to make sure we just get the last one